packages/angularjs-auth-passport/lib/webAuthService.js

```diff
@@ -104,4 +115,16 @@ WebAuthService.prototype.logout = function() {
   });
 }
 
+
+/**
+ * Private - show dialog when cloudUrl is missing
+ */
+WebAuthService.prototype.showAlertDialog = function() {
+  return self.dialog.show(self.dialog.alert({
```

Comment:
uses self but self is not defined

Reply:
+1, didn't catch this, I wonder why eslint didn't too though

Reply:
I see, looks like we don't have eslint setup for some of the packages, I'll add a `npm test` script that at least runs it for the ones that are missing in a separate PR

Reply:
Not sure how this were working on my local machine :D